src/alto-ui/Form/ColorPicker/story.js

```diff
@@ -37,41 +37,34 @@ storiesOf('Form/ColorPicker', module)
   .addDecorator(withReadme(README))
   .addDecorator(centered)
   .addWithJSX('playground', () => (
-    <StateFullColorPicker value="dodgerblue">
+    <StateFullColorPicker value="blue">
       {(value, onChange) => (
         <div style={{ display: 'flex' }}>
-          <ColorPicker
-            name="alto-colorpicker"
-            id="tomato"
-            color="tomato"
-            label="Red"
-            onChange={() => onChange('tomato')}
-            checked={value === 'tomato'}
-          />
-          <ColorPicker
-            name="alto-colorpicker"
-            id="mediumseagreen"
-            color="mediumseagreen"
-            label="Green"
-            onChange={() => onChange('mediumseagreen')}
-            checked={value === 'mediumseagreen'}
-          />
-          <ColorPicker
-            name="alto-colorpicker"
-            id="dodgerblue"
-            color="dodgerblue"
-            label="Blue"
-            onChange={() => onChange('dodgerblue')}
-            checked={value === 'dodgerblue'}
-          />
-          <ColorPicker
-            name="alto-colorpicker"
-            id="darkviolet"
-            color="darkviolet"
-            label="Purple"
-            onChange={() => onChange('darkviolet')}
-            checked={value === 'darkviolet'}
-          />
+          {[
+            'red',
```

Comment:
are these color list going to be static? and are we going to need them in somewhere else in the future?